docs/driver-only-builds.md

```diff
@@ -215,8 +200,7 @@ consequence these are not supported in builds without `MBEDTLS_ECDSA_C`.
 
 Similarly, there is no PSA support for interruptible ECDH operations so these
 are not supported without `ECDH_C`. See also limitations regarding
-restartable operations with `MBEDTLS_USE_PSA_CRYPTO` in [its
-documentation](use-psa-crypto.md).
+restartable operations in [the documentation about using PSA Crypto](use-psa-crypto.md).
```

Comment:
With `use-psa-crypto.md` deleted: In the documentation of `MBEDTLS_ECP_RESTARTABLE`, or in `docs/architecture/psa-migration/psa-limitations.md`.